content/pages/healthcare/apply.md

```diff
@@ -66,7 +66,6 @@ To update your personal, insurance, or financial information from a previous app
 ### Additional forms for caregivers, National Guard, and Reserve Servicemembers
```

Comment:
Are you talking about these lines?

I haven't been following this directly, though I know a link is being removed. Why aren't we replacing it with a different way for National Guard to access an application?

 - ARE YOU A CAREGIVER, NATIONAL GUARD, OR RESERVE SERVICEMEMBER? THIS FORM’S NOT QUITE READY FOR YOU YET. FIND THE FORM YOU NEED.

 - Additional forms for caregivers, National Guard, and Reserve Servicemembers